unittest.TextTestRunner.run return type misjudge

I've got the following error when using mypy:

```
$ mypy -s foo.py
foo.py: note: In function "run_tests":
foo.py:10: error: "run" of "TestRunner" does not return a value
```
However, according to [3.5.2 documentation](https://docs.python.org/3/library/unittest.html#unittest.TestResult):
> ...a TestResult instance is returned by the TestRunner.run() method...

Although the result class can be altered in the constructor of `TextTestRunner`, I believe it should still be a child class of `TestResult`.

It should be able to fix by changing the return value in `typeshed/stdlib/3/unittest/__init__.pyi:290` from `None` to `TestResult`.

--- a/stdlib/3/unittest/__init__.pyi
+++ b/stdlib/3/unittest/__init__.pyi
@@ -285,9 +285,8 @@ defaultTestLoader = ...  # type: TestLoader
 
 _ResultClassType = Callable[[TextIO, bool, int], TestResult]
 
-# not really documented
 class TestRunner:
-    def run(self, test: Union[TestSuite, TestCase]) -> None: ...
+    def run(self, test: Union[TestSuite, TestCase]) -> TestResult: ...
 
 class TextTestRunner(TestRunner):
     if sys.version_info >= (3, 5):

